hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonOutputCommitter.java

```diff
@@ -266,7 +270,16 @@ private void commitJobForPartition(JobContext context, boolean overwriteSet,
         && operationContext != null) {
       uuid = operationContext.getProperty("uuid").toString();
     }
+    String segmentFileName = SegmentFileStore.genSegmentFileName(
+        loadModel.getSegmentId(), String.valueOf(loadModel.getFactTimeStamp()));
+    boolean isMergeIndex = Boolean.parseBoolean(CarbonProperties.getInstance()
```

Comment:
Variable `isMergeIndexEnabled` is already defined in caller method Line No:192. Please move the code before calling `commitJobForPartition` and reuse

Reply:
This issue is applicable only when table is partitioned. I think, it is appropriate to do inside the `commitJobForPartition()`